apps/src/templates/school-association.module.scss

```diff
@@ -9,45 +9,56 @@
   h2 {
     margin-bottom: 0.25rem;
   }
-}
-
-.inputContainer {
-  justify-self: center;
-  text-align: left;
-  width: 400px;
-  display: flex;
-  flex-direction: column;
 
-  & > div > label > * {
+  label {
     width: 100%;
   }
 
-  & > div > label > input {
+  input {
+    width: 100%;
     box-sizing: border-box;
     height: 46px !important;
     font-size: 1rem;
     margin-top: 0.37rem !important;
     border-radius: 0.25rem;
     border: 1px solid;
-    background-color: inherit;
     text-indent: 0.75rem;
   }
 
+  select {
+    font-weight: 400 !important;
+    width: 100% !important;
```

Comment:
Is `!important` needed here in order to override the component library styling? I wonder if it would be better to use classnames to do so?

Reply:
Yes unfortunately. And do we have a norm around using classnames? We moved this one out of a classname to standardize across the components included in this stylesheet.

Reply:
Will other components matter given these are nested in the outer classname .outerContainer? And wouldn't the other components have to import this stylesheet? Maybe @kelbyhawn could be a helpful person to bring in here as well.

Reply:
OH totally missed `.outerContainer` (it was above github's preview 😭)

Reply:
> And wouldn't the other components have to import this stylesheet?

I'm thinking of the case of adding this to https://code.org/yourschool which has other selects. When we update that page, this stylesheet will be brought in.

If this only applies to selects within `.outerContainer`, I think this is fine

Reply:
I suggest renaming the `.outerContainer` class to something more specific like `.schoolAssociationWrapper` just in case!

Reply:
Renamed accordingly! I tested locally to make sure it was all still picked up:)